reinforcement_learning/rl_hvac_coach_energyplus/rl_hvac_coach_energyplus.ipynb

```diff
@@ -294,10 +294,10 @@
     "estimator = RLEstimator(entry_point=\"train-coach.py\",\n",
     "                        source_dir='src',\n",
     "                        dependencies=[\"common/sagemaker_rl\"],\n",
-    "                        image_name=custom_image_name,\n",
+    "                        image_uri=custom_image_name,\n",
```

Comment:
does it require an uri?

Reply:
Its the same parameter, just renamed in SDK v2. `custom_image_name` has always been a docker image URI.